e2e/cypress.json

```diff
@@ -13,10 +13,10 @@
         "adminPassword": "Sys@dmin-sample1",
         "allowedUntrustedInternalConnections": "localhost",
         "cwsUrl": "http://localhost:8076",
-        "cwsApiUrl": "http://localhost:8076",
+        "cwsAPIURL": "http://localhost:8076",
```

Comment:
Should this be changing? It seems like something that might be needed by Cypress itself

Reply:
I wanted to keep the cypress env names in sync with the config names. They are used to communicate values to the cypress tests. AFAIK they are only defined here.